android/PhysicalWeb/app/build.gradle

```diff
@@ -53,7 +53,7 @@ android {
     lintOptions {
         // We'll get to fixing the icon later
         disable 'GoogleAppIndexingWarning', 'IconLauncherShape', 'IconDensities',
-            'IconMissingDensityFolder', 'LongLogTag'
+            'IconMissingDensityFolder', 'LongLogTag', 'OldTargetApi', 'GradleDependency'
```

Comment:
Please add a comment explaining why we suppress these errors so we'll know when/if it's okay to stop suppressing them.